src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -342,35 +343,37 @@ export const MoreActionsDrawer = ({
               </ToggleButton>
             </ListItem>
 
-            <ListItem disablePadding={false}>
+            <ListItem disablePadding>
```

Comment:
By setting `disablePadding` to true, the row now goes all the way to the edge of the drawer, which is a bit awkward:

![image](https://github.com/user-attachments/assets/2dd86289-81c2-4a32-a2ca-b1cf9352d3f3)

We should be keeping this `false`

```suggestion
            <ListItem disablePadding={false}>
```